src/components/App/App.tsx

```diff
@@ -99,11 +93,7 @@ function App() {
       <Route exact path="/:country" render={ ({ match }) => {
         return <QuizPage 
           // {...match}
-          questions={[
```

Comment:
Great job getting these questions moved. makes much more sense to only have them in the quiz component!